bcda/models/models.go

```diff
@@ -80,10 +81,22 @@ type ACOBeneficiary struct {
 	// Join model needed for additional fields later, e.g., AttributionDate
 }
 
+type CCLF9 struct {
+	gorm.Model
+	CurrentNum   string `json:"current_number"`
```

Comment:
The CCLF8 model includes some metadata... I wonder if we need to be storing that here too?
1) ACO ID - do we need to tie this data back to an ACO?
2) File date
3) File performance year
4) Imported date/created at

Reply:
not so sure about the other stuff, but we get the created at and deleted at for free in gorm.

Reply:
that is a DB level created at though.  The CCLF9 has its own date that I think people would be interested in knowing.

Reply:
are u referring to the previous effective and obsolete dates that come in that file? Or the actual delivery date of that file?

Reply:
I think the date embedded in the name of the file.